curriculum/challenges/english/07-scientific-computing-with-python/scientific-computing-with-python-projects/budget-app.md

```diff
@@ -18,7 +18,7 @@ Complete the `Category` class. It should be able to instantiate objects based on
 
 When the budget object is printed it should display:
 
-- A title line of 30 characters where the name of the category is centered in a line of `*` characters.
+- A title line of 30 characters where the name of the category is centered in a line of "*" characters.
```

Comment:
For Python, we are using single quotes instead of double.

```suggestion
- A title line of 30 characters where the name of the category is centered in a line of '*' characters.
```